### PR TITLE
Documents and tweaks term_ping filtering option

### PR DESCRIPTION
**Pull request by penny64**
[QOL]
## About the PR 
This documents the term_ping filtering functionality in the TermOS help command, and adjusts it so that it accepts the first argument as the filter- not the second

Old

`term_ping asdf PNET_MAINFRAME`

New

`term_ping PNET_MAINFRAME`

## Why's this needed? 

Documentation of features in game is good, and taking the first argument is consistent with the other commands like `connect`
I'm not sure taking the second argument was ever actually intended.

## Changelog

```changelog
(u)Penny
(+)The term_ping command now accepts a filter as the first option
(+)The TermOS help command now documents the term_ping filter
```

### CODE REVIEW

File: code/modules/networks/computer3/terminal.dm [hunk 1]
```diff
@@ -96,8 +96,8 @@ file_save - Save file to local disk."}
 							src.net_number = new_net_number
 
 					src.peripheral_command("subnet[src.net_number]", null, "\ref[src.netcard]")
-				if (length(command_list) > 1)
-					src.ping_filter = lowertext(command_list[2])
+				if (length(command_list) > 0)
```
Comment (ZeWaka) [inline code/modules/networks/computer3/terminal.dm:99]:
```suggestion
				if (length(command_list))
```

**Review by ZeWaka — APPROVED** [review]
thanks for doing this!

**Comment by witheld9** [comment]
Glad to help! I don't need to do anything on behalf of your changes right @ZeWaka ?

**Comment by ZeWaka** [comment]
nop!

**Pull request merged**